# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mini_sendmail/install.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- mini_sendmail-1.3.5/Makefile	2002-01-04 06:05:25.000000000 +0000
+++ mini_sendmail-1.3.5-fixed/Makefile	2007-03-27 09:17:29.000000000 +0000
@@ -4,8 +4,10 @@
 # Solaris, you will need to uncomment this definition.
 #SYSV_LIBS =    -lnsl -lsocket
 
-BINDIR =	/usr/local/sbin
-MANDIR =	/usr/local/man
+prefix =	/usr/local
+
+BINDIR =	$(prefix)/sbin
+MANDIR =	$(prefix)/man
 CC =		gcc
 CFLAGS =	-O
 #CFLAGS =	-g
@@ -30,10 +32,10 @@
 
 
 install:	all
-	rm -f $(BINDIR)/mini_sendmail
-	cp mini_sendmail $(BINDIR)
-	rm -f $(MANDIR)/man8/mini_sendmail.8
-	cp mini_sendmail.8 $(MANDIR)/man8
+	rm -f $(DESTDIR)$(BINDIR)/mini_sendmail
+	cp mini_sendmail $(DESTDIR)$(BINDIR)
+	rm -f $(DESTDIR)$(MANDIR)/man8/mini_sendmail.8
+	cp mini_sendmail.8 $(DESTDIR)$(MANDIR)/man8
 
 clean:
 	rm -f mini_sendmail *.o core core.* *.core
